tests/integration/routes/accounts/delete-accounts-test.js

```diff
@@ -91,58 +91,80 @@ getServer(function (error, server) {
       t.end()
     })
 
-    group.test('Not an admin', {todo: true}, function (t) {
-      t.end()
-    })
+    // group.test('Not an admin', {todo: false}, function (t) {
+
+    //   t.end()
+
+    // })
+
+      //EXAMPLE
+      group.test('Not an admin', {todo: true}, function (t) {
+        server.inject({
+          method: 'DELETE',
+          url: '/accounts/{id}',
+          headers: {
+            accept: 'application/vnd.api+json',
+            authorization: 'Session cGF0LWRvZToxMjc1MDA6zEZsQ1BuO-W8SthDSrg8KXQ8OlQ',
+            'content-type': 'application/vnd.api+json'
+          }
+        }, function (response) {
+          t.is(response.statusCode, 401, 'returns 401 status')
+          t.is(response.result.errors[0].title, 'Unauthorized', 'returns "Unauthorized" error')
+          t.is(response.result.errors[0].detail, 'Session invalid', 'returns "Session invalid" error')
+          t.end()
+        })
+      })
+      //END OF TEMP EXAPMLE
```

Comment:
test looks good 👍 

I found that in the `DELETE /accounts/{id}` handler we don’t even check for an admin session: 
https://github.com/hoodiehq/hoodie-account-server/blob/master/routes/accounts.js#L212-L226

Compare to what we do in the [postAccountsRoute handler](https://github.com/hoodiehq/hoodie-account-server/blob/master/routes/accounts.js#L38). Also check the [error handling](https://github.com/hoodiehq/hoodie-account-server/blob/master/routes/accounts.js#L60-L65)